source/extensions/filters/http/oauth2/filter.h

```diff
@@ -134,6 +134,8 @@ class FilterConfig : public Logger::Loggable<Logger::Id::oauth2> {
                const std::string& stats_prefix);
   const std::string& clusterName() const { return oauth_token_endpoint_.cluster(); }
   const std::string& clientId() const { return client_id_; }
+
+  const TokenType& forwardBearerTokenType() const { return forward_bearer_token_type_; }
```

Comment:
```suggestion
TokenType forwardBearerTokenType() const { return forward_bearer_token_type_; }
```
We needn't const reference to an enum